From ebf1da48a73f25075b20ea8d3354b3f89e68436e Mon Sep 17 00:00:00 2001
From: Yinbo Zhu <yinbo.zhu@nxp.com>
Date: Mon, 10 Dec 2018 16:12:43 +0800
Subject: [PATCH 736/767] usb: dwc3/host: fix compile warning issue by
 repairing usb module

this patch is to replace u32 out-put format with u64

Signed-off-by: Yinbo Zhu <yinbo.zhu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.12.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/usb/host/xhci.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/xhci.c b/drivers/usb/host/xhci.c
index 0ff9ae6f8fad..8bfb2a8b7053 100644
--- a/drivers/usb/host/xhci.c
+++ b/drivers/usb/host/xhci.c
@@ -5119,7 +5119,7 @@ int xhci_gen_setup(struct usb_hcd *hcd, xhci_get_quirks_t get_quirks)
 		return retval;
 	xhci_dbg(xhci, "Called HCD init\n");
 
-	xhci_info(xhci, "hcc params 0x%08x hci version 0x%x quirks 0x%016llx\n",
+	xhci_info(xhci, "hcc params 0x%08x hci version 0x%x quirks 0x%llx\n",
 		  xhci->hcc_params, xhci->hci_version, xhci->quirks);
 
 	return 0;
-- 
2.17.0

